packages/L10n/src/useI18n.js

```diff
@@ -3,8 +3,24 @@ import L10nContext from "./L10nContext";
 import { i18n } from "./i18n";
 import PaprikaLocales from "./locales";
 
-export const getI18nObject = (locale = "en", Locales = PaprikaLocales) => {
-  const _i18n = i18n(Locales);
+function mergeTranslation(customLocales, paprikaLocales) {
+  const result = {};
+
+  Object.keys(paprikaLocales).forEach(lang => {
+    result[lang] = {
+      translation: {
+        ...(customLocales[lang] ? customLocales[lang].translation : {}),
+        ...(paprikaLocales[lang] ? paprikaLocales[lang].translation : {}),
```

Comment:
Will this make Paprika override what the user provides?  I'd think it should be the other way around.

Reply:
Right, I'll add notes in the readme as well